 
Data Hora: 12/03/2018 15:3:51
Rotina:    System.Data.Common.DbDataReader ExecuteStoreCommands(System.Data.Entity.Core.EntityClient.EntityCommand, System.Data.CommandBehavior)
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: ERRO_EntityCommandExecutionException
Título:    Já existe um DataReader aberto associado a este Command que deve ser fechado primeiro.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.Internal.EntityCommandDefinition.ExecuteStoreCommands(EntityCommand entityCommand, CommandBehavior behavior)
   em System.Data.Entity.Core.Objects.Internal.ObjectQueryExecutionPlan.Execute[TResultType](ObjectContext context, ObjectParameterCollection parameterValues)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__a()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.ParametroService.Get(String pParametro, Nullable`1 pIdFilial) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\ParametroService.cs:linha 70
   em FrameWork.Excecao.RepositorioChaves.TNucleoPainel.GetTempoExpiracao(Int32 idFIlial) na D:\DESENVOLVIMENTO\Projetos\PLUS\Framework\NucleoCompartilhado\RepositorioChaves\TNucleoPainel.cs:linha 52
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 128

--------------------------------------------
 
Data Hora: 12/03/2018 18:6:20
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/integracao/empresa/filiais
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Controllers.EmpresaController.GetFiliais(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Integracao\EmpresaController.cs:linha 66

--------------------------------------------
 
Data Hora: 12/03/2018 18:6:37
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 12/03/2018 18:7:4
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/integracao/empresa/filiais
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Controllers.EmpresaController.GetFiliais(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Integracao\EmpresaController.cs:linha 66

--------------------------------------------
 
Data Hora: 12/03/2018 18:7:5
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 12/03/2018 18:7:53
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 12/03/2018 18:8:29
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/integracao/empresa/filiais
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Controllers.EmpresaController.GetFiliais(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Integracao\EmpresaController.cs:linha 66

--------------------------------------------
 
Data Hora: 12/03/2018 18:8:29
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 12/03/2018 22:3:34
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/integracao/empresa/filiais
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Controllers.EmpresaController.GetFiliais(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Integracao\EmpresaController.cs:linha 66

--------------------------------------------
 
Data Hora: 12/03/2018 22:3:51
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 12/03/2018 22:4:32
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/integracao/empresa/filiais
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Controllers.EmpresaController.GetFiliais(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Integracao\EmpresaController.cs:linha 66

--------------------------------------------
 
Data Hora: 12/03/2018 22:4:49
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 12/03/2018 22:6:45
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 12/03/2018 22:6:50
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 12/03/2018 22:9:4
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/integracao/empresa/filiais
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Controllers.EmpresaController.GetFiliais(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Integracao\EmpresaController.cs:linha 66

--------------------------------------------
 
Data Hora: 12/03/2018 22:9:21
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 12/03/2018 22:9:59
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/integracao/empresa/filiais
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Controllers.EmpresaController.GetFiliais(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Integracao\EmpresaController.cs:linha 66

--------------------------------------------
 
Data Hora: 12/03/2018 22:10:27
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/integracao/empresa/filiais
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Controllers.EmpresaController.GetFiliais(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Integracao\EmpresaController.cs:linha 66

--------------------------------------------
 
Data Hora: 12/03/2018 22:10:48
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 12/03/2018 22:13:28
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/integracao/empresa/filiais
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Controllers.EmpresaController.GetFiliais(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Integracao\EmpresaController.cs:linha 66

--------------------------------------------
 
Data Hora: 12/03/2018 22:13:45
Rotina:    Void Open()
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    The underlying provider failed on Open.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 99

--------------------------------------------
 
Data Hora: 12/03/2018 22:17:12
Rotina:    System.String GetProviderManifestTokenChecked(System.Data.Entity.Core.Common.DbProviderServices, System.Data.Common.DbConnection)
Rota:      http://localhost:2020/api/v1/integracao/empresa/filiais
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: ERRO_ProviderIncompatibleException
Título:    Erro de rede ou específico à instância ao estabelecer conexão com o SQL Server. O servidor não foi encontrado ou não estava acessível. Verifique se o nome da instância está correto e se o SQL Server está configurado para permitir conexões remotas. (provider: Named Pipes Provider, error: 40 - Não foi possível abrir uma conexão com o SQL Server)
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.Entity.Utilities.DbProviderServicesExtensions.GetProviderManifestTokenChecked(DbProviderServices providerServices, DbConnection connection)
   em System.Data.Entity.Infrastructure.DefaultManifestTokenResolver.<>c__DisplayClass1.<ResolveManifestToken>b__0(Tuple`3 k)
   em System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
   em System.Data.Entity.Infrastructure.DefaultManifestTokenResolver.ResolveManifestToken(DbConnection connection)
   em System.Data.Entity.Utilities.DbConnectionExtensions.GetProviderInfo(DbConnection connection, DbProviderManifest& providerManifest)
   em System.Data.Entity.DbModelBuilder.Build(DbConnection providerConnection)
   em System.Data.Entity.Internal.LazyInternalContext.CreateModel(LazyInternalContext internalContext)
   em System.Data.Entity.Internal.RetryLazy`2.GetValue(TInput input)
   em System.Data.Entity.Internal.LazyInternalContext.InitializeContext()
   em System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   em System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   em System.Data.Entity.Internal.Linq.InternalSet`1.Include(String path)
   em System.Data.Entity.Infrastructure.DbQuery`1.Include(String path)
   em PainelControle.ApplicationService.Service.UsuarioService.Get(String login, Boolean usoInterno) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\UsuarioService.cs:linha 112
   em Plus.Api.Controllers.EmpresaController.GetFiliais(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Integracao\EmpresaController.cs:linha 66

--------------------------------------------
 
Data Hora: 12/03/2018 22:26:43
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/faturamento/documentosaida/dicionario/
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.Faturamento.DocumentosaidaController.GetDicionario(HttpRequestMessage request) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Faturamento\DocumentosaidaController.cs:linha 47

--------------------------------------------
 
Data Hora: 12/03/2018 23:4:0
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/estoque/produto/produtograde?$orderby=codigo&$select=descricaoCompleta,descricao,codigo,unidadeComercial,ean,codigoAuxiliar,descricaoGrupo,id&$filter=contains(descricaoCompleta,%27CAMIS%27)%20or%20contains(ean,%27CAMIS%27)%20or%20contains(codigoAuxiliar,%27CAMIS%27)%20or%20contains(codigo,%27CAMIS%27)&$expand=estProduto($expand=estProdutoGrupo,estProdutoImposto($expand=estoqueEstArmazem,fisModelotributacao($expand=fisModelotributacaoIcms))),estProdutoGrade,estProdutoKit($expand=estProduto($expand=estProdutoImposto($expand=estoqueEstArmazem)))
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     99
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 99
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.ProdutoController.Get(HttpRequestMessage request, ODataQueryOptions`1 opts) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Estoque\ProdutoController.cs:linha 249

--------------------------------------------
 
Data Hora: 12/03/2018 23:4:1
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/estoque/produto/produtograde?$orderby=codigo&$select=descricaoCompleta,descricao,codigo,unidadeComercial,ean,codigoAuxiliar,descricaoGrupo,id&$filter=contains(descricaoCompleta,%27CAM%27)%20or%20contains(ean,%27CAM%27)%20or%20contains(codigoAuxiliar,%27CAM%27)%20or%20contains(codigo,%27CAM%27)&$expand=estProduto($expand=estProdutoGrupo,estProdutoImposto($expand=estoqueEstArmazem,fisModelotributacao($expand=fisModelotributacaoIcms))),estProdutoGrade,estProdutoKit($expand=estProduto($expand=estProdutoImposto($expand=estoqueEstArmazem)))
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.ProdutoController.Get(HttpRequestMessage request, ODataQueryOptions`1 opts) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Estoque\ProdutoController.cs:linha 249

--------------------------------------------
 
Data Hora: 12/03/2018 23:4:1
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/estoque/produto/produtograde?$orderby=codigo&$select=descricaoCompleta,descricao,codigo,unidadeComercial,ean,codigoAuxiliar,descricaoGrupo,id&$filter=contains(descricaoCompleta,%27CAMIS%27)%20or%20contains(ean,%27CAMIS%27)%20or%20contains(codigoAuxiliar,%27CAMIS%27)%20or%20contains(codigo,%27CAMIS%27)&$expand=estProduto($expand=estProdutoGrupo,estProdutoImposto($expand=estoqueEstArmazem,fisModelotributacao($expand=fisModelotributacaoIcms))),estProdutoGrade,estProdutoKit($expand=estProduto($expand=estProdutoImposto($expand=estoqueEstArmazem)))
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.ProdutoController.Get(HttpRequestMessage request, ODataQueryOptions`1 opts) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Estoque\ProdutoController.cs:linha 249

--------------------------------------------
 
Data Hora: 12/03/2018 23:4:16
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/estoque/produto/produtograde?$orderby=codigo&$select=descricaoCompleta,descricao,codigo,unidadeComercial,ean,codigoAuxiliar,descricaoGrupo,id&$filter=contains(descricaoCompleta,%27ARR%27)%20or%20contains(ean,%27ARR%27)%20or%20contains(codigoAuxiliar,%27ARR%27)%20or%20contains(codigo,%27ARR%27)&$expand=estProduto($expand=estProdutoGrupo,estProdutoImposto($expand=estoqueEstArmazem,fisModelotributacao($expand=fisModelotributacaoIcms))),estProdutoGrade,estProdutoKit($expand=estProduto($expand=estProdutoImposto($expand=estoqueEstArmazem)))
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.ProdutoController.Get(HttpRequestMessage request, ODataQueryOptions`1 opts) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Estoque\ProdutoController.cs:linha 249

--------------------------------------------
 
Data Hora: 12/03/2018 23:4:17
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/estoque/produto/produtograde?$orderby=codigo&$select=descricaoCompleta,descricao,codigo,unidadeComercial,ean,codigoAuxiliar,descricaoGrupo,id&$filter=contains(descricaoCompleta,%27ARRO%27)%20or%20contains(ean,%27ARRO%27)%20or%20contains(codigoAuxiliar,%27ARRO%27)%20or%20contains(codigo,%27ARRO%27)&$expand=estProduto($expand=estProdutoGrupo,estProdutoImposto($expand=estoqueEstArmazem,fisModelotributacao($expand=fisModelotributacaoIcms))),estProdutoGrade,estProdutoKit($expand=estProduto($expand=estProdutoImposto($expand=estoqueEstArmazem)))
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.ProdutoController.Get(HttpRequestMessage request, ODataQueryOptions`1 opts) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Estoque\ProdutoController.cs:linha 249

--------------------------------------------
 
Data Hora: 12/03/2018 23:4:17
Rotina:    Void LoginUnico(PainelControle.ApplicationService.IServices.IUsuarioService)
Rota:      http://localhost:2020/api/v1/estoque/produto/produtograde?$orderby=codigo&$select=descricaoCompleta,descricao,codigo,unidadeComercial,ean,codigoAuxiliar,descricaoGrupo,id&$filter=contains(descricaoCompleta,%27ARROZ%27)%20or%20contains(ean,%27ARROZ%27)%20or%20contains(codigoAuxiliar,%27ARROZ%27)%20or%20contains(codigo,%27ARROZ%27)&$expand=estProduto($expand=estProdutoGrupo,estProdutoImposto($expand=estoqueEstArmazem,fisModelotributacao($expand=fisModelotributacaoIcms))),estProdutoGrade,estProdutoKit($expand=estProduto($expand=estProdutoImposto($expand=estoqueEstArmazem)))
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     106
Tipo Erro: 1
Nome Erro: TOK0001
Título:    Acesso Duplicado!
Motivo:    Seu login de acesso já se encontra em uso em outro(s) terminal(is)!
Solução :  Efetue o logout em todos os outros terminais.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.LoginUnico(IUsuarioService usuarioService) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 106
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 32
   em Plus.Api.Controllers.ProdutoController.Get(HttpRequestMessage request, ODataQueryOptions`1 opts) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\Estoque\ProdutoController.cs:linha 249

--------------------------------------------
 
Data Hora: 12/03/2018 23:6:41
Rotina:    Void OnError(System.Data.SqlClient.SqlException, Boolean, System.Action`1[System.Action])
Rota:      http://localhost:2020/api/v1/seguranca/token
FileName:  
Linha:     0
Tipo Erro: 1
Nome Erro: 
Título:    Tempo Limite de Execução Expirado.  O período de tempo limite terminou antes da conclusão da operação ou o servidor não está respondendo.
Motivo:    Erro de sistema
Solução :  Contate o Suporte!
-----------------TRACE----------------------
   em System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   em System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   em System.Data.SqlClient.TdsParserStateObject.ReadSniError(TdsParserStateObject stateObj, UInt32 error)
   em System.Data.SqlClient.TdsParserStateObject.ReadSniSyncOverAsync()
   em System.Data.SqlClient.TdsParserStateObject.TryReadNetworkPacket()
   em System.Data.SqlClient.TdsParserStateObject.TryPrepareBuffer()
   em System.Data.SqlClient.TdsParserStateObject.TryReadByteArray(Byte[] buff, Int32 offset, Int32 len, Int32& totalRead)
   em System.Data.SqlClient.TdsParserStateObject.TryReadStringWithEncoding(Int32 length, Encoding encoding, Boolean isPlp, String& value)
   em System.Data.SqlClient.TdsParser.TryReadSqlStringValue(SqlBuffer value, Byte type, Int32 length, Encoding encoding, Boolean isPlp, TdsParserStateObject stateObj)
   em System.Data.SqlClient.TdsParser.TryReadSqlValue(SqlBuffer value, SqlMetaDataPriv md, Int32 length, TdsParserStateObject stateObj, SqlCommandColumnEncryptionSetting columnEncryptionOverride, String columnName)
   em System.Data.SqlClient.SqlDataReader.TryReadColumnInternal(Int32 i, Boolean readHeaderOnly)
   em System.Data.SqlClient.SqlDataReader.TryReadColumn(Int32 i, Boolean setTimeout, Boolean allowPartiallyReadColumn)
   em System.Data.SqlClient.SqlDataReader.GetValues(Object[] values)
   em System.Data.Entity.Core.Objects.Internal.ShapelessBufferedDataRecord.Initialize(String providerManifestToken, DbProviderServices providerSerivces, DbDataReader reader)
   em System.Data.Entity.Core.Objects.Internal.BufferedDataReader.Initialize(String providerManifestToken, DbProviderServices providerServices, Type[] columnTypes, Boolean[] nullableColumns)
   em System.Data.Entity.Core.Objects.Internal.ObjectQueryExecutionPlan.Execute[TResultType](ObjectContext context, ObjectParameterCollection parameterValues)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__a()
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClassb.<GetResults>b__9()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   em System.Lazy`1.CreateValue()
   em System.Lazy`1.LazyInitValue()
   em System.Lazy`1.get_Value()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.<GetElementFunction>b__1[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Data.Entity.Internal.Linq.DbQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.FirstOrDefault[TSource](IQueryable`1 source)
   em PainelControle.ApplicationService.Service.ParametroService.Get(String pParametro, Nullable`1 pIdFilial) na C:\Users\jnunes\Desktop\agent\_work\20\s\PainelControle.ApplicationService\Service\ParametroService.cs:linha 70
   em FrameWork.Excecao.RepositorioChaves.TNucleoPainel.GetTempoExpiracao(Int32 idFIlial) na D:\DESENVOLVIMENTO\Projetos\PLUS\Framework\NucleoCompartilhado\RepositorioChaves\TNucleoPainel.cs:linha 52
   em Plus.Api.Security.AuthorizationServerProvider.<GrantResourceOwnerCredentials>d__7.MoveNext() na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\AuthorizationServerProvider.cs:linha 128

--------------------------------------------
 
Data Hora: 12/03/2018 23:6:49
Rotina:    Void VerificaPermissao(System.String)
Rota:      http://localhost:2020/api/v1/painelControle/parametro/terminal?$select=id,intEmpresaId,parametro,descricao,dominio,valor
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     81
Tipo Erro: 1
Nome Erro: PCT0003
Título:    Sem Permissão!
Motivo:    Este usuário Não tem previlégios para executar esta ação!
Solução :  Contacte o Administrador do sistema.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.VerificaPermissao(String rotina) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 81
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 40
   em Plus.Api.Controllers.ParametroController.GetByTerminal(HttpRequestMessage request, ODataQueryOptions`1 opts) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\PainelControle\ParametroController.cs:linha 71

--------------------------------------------
 
Data Hora: 12/03/2018 23:12:25
Rotina:    Void VerificaPermissao(System.String)
Rota:      http://localhost:2020/api/v1/painelControle/parametro/terminal?$select=id,intEmpresaId,parametro,descricao,dominio,valor
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     81
Tipo Erro: 1
Nome Erro: PCT0003
Título:    Sem Permissão!
Motivo:    Este usuário Não tem previlégios para executar esta ação!
Solução :  Contacte o Administrador do sistema.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.VerificaPermissao(String rotina) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 81
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 40
   em Plus.Api.Controllers.ParametroController.GetByTerminal(HttpRequestMessage request, ODataQueryOptions`1 opts) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\PainelControle\ParametroController.cs:linha 71

--------------------------------------------
 
Data Hora: 12/03/2018 23:17:57
Rotina:    Void VerificaPermissao(System.String)
Rota:      http://localhost:2020/api/v1/painelControle/parametro/terminal?$select=id,intEmpresaId,parametro,descricao,dominio,valor
FileName:  D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs
Linha:     81
Tipo Erro: 1
Nome Erro: PCT0003
Título:    Sem Permissão!
Motivo:    Este usuário Não tem previlégios para executar esta ação!
Solução :  Contacte o Administrador do sistema.
-----------------TRACE----------------------
   em Plus.Api.Security.TControleAcesso.VerificaPermissao(String rotina) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 81
   em Plus.Api.Security.TControleAcesso.Executa(IPrincipal user, String rotina, String dominio, Boolean verificaParametro, Boolean loginUnico, Boolean verificaPermissao) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Security\ControleAcesso.cs:linha 40
   em Plus.Api.Controllers.ParametroController.GetByTerminal(HttpRequestMessage request, ODataQueryOptions`1 opts) na D:\DESENVOLVIMENTO\Projetos\PLUS\Api\Plus.Api\Controllers\PainelControle\ParametroController.cs:linha 71

--------------------------------------------
